scalameter-core/src/main/scala/org/scalameter/utils/ClassPath.scala

```diff
@@ -20,6 +20,12 @@ class ClassPath private (val paths: Seq[File]) extends Serializable {
     new ClassPath(location +: paths)
   }
 
+  /** Returns classpath without first location.
```

Comment:
Then it throws an exception (should I update comment?). I know, not very good, but this method should be used only for cleaning context after adding instrumented jar. 
So maybe limiting access scope could be a good idea?